### PR TITLE
fix: add warning when configuration may lead to use of development ap…

### PR DESCRIPTION
**Pull request by Nebulis**
…i key

### CODE REVIEW

File: src/verifiers/verificationBuilder.ts [hunk 1]
```diff
@@ -8,18 +8,26 @@ import {
...
  */
+let displayWarning = true;
 export const verificationBuilder = <T extends Verifier<any>>(
   verifiers: T[],
   builderOptions: VerificationBuilderOptions
 ) => (document: DocumentsToVerify, promisesCallback?: PromiseCallback): Promise<VerificationFragment[]> => {
+  // if the user didn't configure an API key and didn't configure a provider or a resolver, then he will likely use a development key. We then warn him once, that he may need to configure things properly, especially for production
+  if (displayWarning && (!builderOptions.resolver || !builderOptions.provider) && !process.env.INFURA_API_KEY) {
+    displayWarning = false;
+    console.warn(
```
Comment (rjchow) [inline src/verifiers/verificationBuilder.ts:27]: oa-verify doesn't have debug?

**Review by rjchow — APPROVED** [review]

File: src/verifiers/verificationBuilder.ts [hunk 2]
```diff
@@ -8,18 +8,26 @@ import {
...
+      "You are using oa-verify default configuration, which is not suitable for production environment. Please make sure that yu configured the library correctly."
```
Comment (SiaChenRen) [inline src/verifiers/verificationBuilder.ts:28]: Have abit typo 
"Please make sure that yu configured the library correctly"
to 
"Please make sure that you configured the library correctly"

File: src/verifiers/verificationBuilder.ts (hunk 1 above)
Comment (Nebulis) [reply src/verifiers/verificationBuilder.ts:27]: which means need to setup log level. Nah I want this one to be visible

File: src/verifiers/verificationBuilder.ts (hunk 2 above)
Comment (Nebulis) [reply src/verifiers/verificationBuilder.ts:28]: thanks

File: src/verifiers/verificationBuilder.ts (hunk 1 above)
Comment (rjchow) [reply src/verifiers/verificationBuilder.ts:27]: ok

**Pull request merged**